src/main/java/DeleteCommand.java

```diff
@@ -0,0 +1,37 @@
+import java.util.ArrayList;
+
+public class DeleteCommand extends Command{
+    static final int commandLength = 2;
+    static final int indexOfIndex = 1;
```

Comment:
As these 2 variables are final, should consider probably fully capitalising them like COMMAND_LENGTH for example.